app/src/main/res/layout/activity_help_parameters.xml

```diff
@@ -20,10 +20,13 @@
         android:layout_marginTop="2dp"
         android:layout_marginEnd="2dp"
         android:layout_marginBottom="60dp"
-        android:background="@drawable/select_meds_button"
         android:checked="false"
         android:filterTouchesWhenObscured="false"
         android:textOff="@string/first_aid_kit"
+        android:background="@drawable/first_aid_kit"
```

Comment:
There is no more feedback on which medication is selected. 

It would be cool that the selected one could be enligthed maybe

Reply:
You're indeed correct, I didn't pay attention to this issue. However, I couldn't find a way to highlight the button, the only relevant methods I found were changing the background or text color on selection, both of which are not really a solution. I changed the text to "Selected" once the button is selected and changed the activity's code accordingly.